src/ceph_osd.cc

```diff
@@ -129,13 +129,8 @@ int main(int argc, const char **argv)
     exit(0);
   }
 
-  map<string,string> defaults = {
-    // We want to enable leveldb's log, while allowing users to override this
-    // option, therefore we will pass it as a default argument to global_init().
-    { "leveldb_log", "" }
```

Comment:
Does RocksDB have any compatibility with these old options? I assume it has not but prefer to be paranoid here (this applies to the other place we're dealing with opts, BTW).

Reply:
no, and there do not need the old options.

Reply:
ACK, THX.